src/CRISP.BackendChallenge/Repository/ContextRepository.cs

```diff
@@ -18,38 +19,69 @@ public IQueryable<T> Query()
     }
 
     /// <inheritdoc />
-    public IEnumerable<T> GetAll()
+    public IEnumerable<Employee> GetAll()
     {
-        throw new NotImplementedException();
+        var entities = _context.Employees.Include(e => e.Logins).ToList();
+        if (entities == null)
+        {
+            throw new ArgumentNullException();
```

Comment:
I made a similar comment earlier, but this is not good practice for two reasons:

1. This is not what `ArgumentNullException` is used for 
2. You're purposefully throwing a brand new exception (expensive thing to do), rather than just passing back the null value (still not ideal but much more performant and less code than an exception)

Reply:
Makes sense, fixed those excpetions and returning null instead